### ISSUE TITLE
Trailing comma on JSON objects not allowed

### ISSUE THREAD

**Issue by oscarolar** [OCA/pylint-odoo#126]
Some browsers are able to parse correctly the following:
```javascript
        [{
            trigger: "a:contains(EN)",
            content: "Change Lang to EN",   
        },
        {
            trigger: "a:contains(ES)",
            content: "Change Lang to ES",   
            extra_trigger: "a[data-lang=en_US].active",
        },]
```
but the sad thing is we have to do it for all the browsers so the correct syntax would be:
```javascript
        [{
            trigger: "a:contains(EN)",
            content: "Change Lang to EN",   
        },
        {
            trigger: "a:contains(ES)",
            content: "Change Lang to ES",   
            extra_trigger: "a[data-lang=en_US].active",
        }]
```
this is allowed from [JSON5](http://json5.org/)(specified in ES5) towards so the phantom browser does not handle it and errors can be caused on tests.

**Comment by moylop260** [comment]
FYI eslint has the following check but is not just for last item of a `{}` item.
This will show 3 errors for your example: 

```js
        [{
            trigger: "a:contains(EN)",
            content: "Change Lang to EN",  # <<<---- no-comma-dangle
        },
        {
            trigger: "a:contains(ES)",
            content: "Change Lang to ES",   
            extra_trigger: "a[data-lang=en_US].active",   # <<<---- no-comma-dangle
        },]   # <<<---- no-comma-dangle
```

More info about: http://eslint.org/docs/rules/no-comma-dangle

Do you want enabled?

**Comment by oscarolar** [comment]
totally

**Comment by moylop260** [comment]
PR Related: https://github.com/OCA/pylint-odoo/issues/127
